mscore/prefsdialog.cpp

```diff
@@ -559,6 +569,22 @@ PreferenceDialog::~PreferenceDialog()
       qDeleteAll(localShortcuts);
       }
 
+//---------------------------------------------------------
+//   retranslate
+//---------------------------------------------------------
+
+void PreferenceDialog::retranslate()
+      {
+      retranslateUi(this);
+#ifdef Q_OS_MAC // On Mac, we have an extra theme option,
+      // namely, to follow the system's Dark Mode.
+      // Of course, we need to translate that too :)
+      if (CocoaBridge::isSystemDarkModeSupported())
+            styleName->setItemText(2, QCoreApplication::translate("PrefsDialogBase", "System", nullptr));
```

Comment:
the "disambiguation" parameter of QCoreApplication::translate method is optional, you can ignore it

Reply:
Fixed!